pkg/controller/v1alpha2/experiment/util/webhook_util.go

```diff
@@ -22,11 +22,13 @@ import (
 	"fmt"
 	logger "log"
 
-	commonv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/common/v1alpha2"
-	ep_v1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/experiment/v1alpha2"
 	batchv1beta "k8s.io/api/batch/v1beta1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	commonapi "github.com/kubeflow/katib/pkg/api/operators/apis/common/v1alpha2"
+	ep_v1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/experiment/v1alpha2"
```

Comment:
How about experimentsv1alpha2 and commonapiv1alpha2

Reply:
done